src/test/java/org/commcare/formplayer/services/MediaMetaDataServiceTest.java

```diff
@@ -1,83 +1,146 @@
 package org.commcare.formplayer.services;
 
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.when;
+
 import org.commcare.formplayer.objects.MediaMetadataRecord;
 import org.commcare.formplayer.repo.MediaMetaDataRepo;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
-import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.FilterType;
-import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
-
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
 import java.util.UUID;
 
 /**
  * Tests for {@link MediaMetaDataService}
  */
 
-@DataJpaTest
-@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
-@EnableJpaAuditing
-@ComponentScan(
-        basePackageClasses = {MediaMetaDataService.class},
-        useDefaultFilters = false,
-        includeFilters = @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = {
-                MediaMetaDataService.class})
-)
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration
 public class MediaMetaDataServiceTest {
 
     @Autowired
     MediaMetaDataService mediaMetaDataService;
-
-    @Autowired
-    MediaMetaDataRepo mediaMetaDataRepo;
-
     MediaMetadataRecord mediaMetaData;
+    @Autowired
+    private MediaMetaDataRepo mediaMetaDataRepo;
+    private String fileId;
 
-    private String mediaId;
+    private File testFile;
 
     @BeforeEach
-    public void setUp() {
-        mediaId = UUID.randomUUID().toString();
+    public void setUp() throws IOException {
+        fileId = UUID.randomUUID().toString();
+        testFile = new File("media/" + fileId + ".jpg");
+        testFile.createNewFile();
         mediaMetaData = new MediaMetadataRecord(
-                mediaId,
-                "filePath",
+                fileId,
+                "media/" + fileId + ".jpg",
```

Comment:
nit: use a variable to store filePath in and use it here and above while creating file.